api/src/main/java/org/apache/commons/rdf/api/RDFSyntax.java

```diff
@@ -31,7 +31,7 @@
  * and <a href="https://en.wikipedia.org/wiki/TriX_%28syntax%29">TriX</a>.
  * 
  * @see <a href="https://www.w3.org/TR/rdf11-primer/#section-graph-syntax">RDF 1.1 Primer</a>
- *
+ * @see org.apache.commons.rdf.experimental.RDFParser
  */
 public enum RDFSyntax {
```

Comment:
Raised in [COMMONSRDF-47](https://issues.apache.org/jira/browse/COMMONSRDF-47)

It was made an enum as it is documented as the RDF 1.1 serializations, which is constant. But I agree an interface would be better - that could also simplify the `RDFParser` interface which now needs to have a `String` as a fallback.

I think we can change it so it is somewhat compile-compatible with 0.2.0  (e.g. `RDFSyntax x = RDFSyntax.TURTLE`) even if not binary compatible (enums are kept as numbers) - it is 
mainly useful with the Parser interface which we are adding now in upcoming 0.3.0.